test/jdk/java/awt/Graphics/TextAAHintsTest.java

```diff
@@ -139,11 +163,85 @@ public Dimension getPreferredSize() {
         return new Dimension(500,300);
     }
 
-    public static void main(String[] args) throws Exception {
+    public static void createTestUI() {
+        frame = new Frame("Composite and Text Test");
+        TextAAHintsTest textAAHintsTestObject = new TextAAHintsTest();
+        frame.add(textAAHintsTestObject, BorderLayout.NORTH);
+
+        String instructions = """
+                Note: Texts are rendered with different TEXT_ANTIALIASING &
+                   VALUE_TEXT_ANTIALIAS. Text should be B&W, grayscale, and LCD.
+                   Note: The results may be visually the same.
+                1. Verify that first set of text are rendered correctly.
+                2. Second set of text are created using BufferedImage of the first text.
+                3. Third set of text are created using VolatileImage of the first text.
+                """;
+        TextArea instructionTextArea = new TextArea(instructions, 8, 50);
+        instructionTextArea.setEnabled(false);
+        frame.add(instructionTextArea, BorderLayout.CENTER);
+
+        Panel controlPanel = new Panel();
+        Button passButton = new Button("Pass");
+        passButton.addActionListener(e->{
+            testPassed = true;
+            isInterrupted = true;
+            mainThread.interrupt();
+        });
+        Button failButton = new Button("Fail");
+        failButton.addActionListener(e->{
+
+            // Show dialog to read why the testcase was failed and append the
+            // testcase failure reason to the output
+           final Dialog dialog = new Dialog(frame , "TestCase" +
+                    " failure reason", true);
+            TextArea textArea = new TextArea("", 5,60, TextArea.SCROLLBARS_BOTH);
+            dialog.add(textArea, BorderLayout.CENTER);
+
+            Button okButton = new Button("OK");
+            okButton.addActionListener(e1->{
+                failureReason = textArea.getText();
+                dialog.dispose();
+            });
+            Panel ctlPanel = new Panel();
+            ctlPanel.add(okButton);
+            dialog.add(ctlPanel, BorderLayout.SOUTH);
+            dialog.setLocationRelativeTo(null);
+            dialog.pack();
+            dialog.setVisible(true);
+
+            testPassed = false;
+            isInterrupted = true;
+            mainThread.interrupt();
+        });
+        controlPanel.add(passButton);
+        controlPanel.add(failButton);
+        frame.add(controlPanel, BorderLayout.SOUTH);
+        frame.pack();
+        frame.setLocationRelativeTo(null);
+        frame.setVisible(true);
+    }
+
+    public static void main(String[] args) throws InterruptedException, InvocationTargetException {
+        createTestUI();
+        mainThread = Thread.currentThread();
+        try {
+            mainThread.sleep(testTimeOut);
```

Comment:
```suggestion
            Thread.sleep(testTimeOut);
```
`sleep` is a static method, it's better to call it via the class name rather than an instance variable. This will resolve the IDE warning.

Reply:
Replaced Thread with CountDownLatch